afk-daily.sh

```diff
@@ -1,1610 +1,1850 @@
 #!/system/bin/sh
-
-# --- Variables --- #
+# ##############################################################################
+# Script Name   : afk-daily.sh
+# Description   : Script automating daily
+# Args          : <SCREENSHOTLOCATION> <forceFightCampaign> <forceWeekly> <testServer>
+# GitHub        : https://github.com/zebscripts/AFK-Daily
+# License       : MIT
+# ##############################################################################
+
+# ##############################################################################
+# Section       : Variables
+# ##############################################################################
 # Probably you don't need to modify this. Do it if you know what you're doing, I won't blame you (unless you blame me).
 DEVICEWIDTH=1080
-pvpEvent=false # Set to `true` if "Heroes of Esperia" event is live
+DEBUG=0
+# DEBUG  = 0    Show no debug
+# DEBUG >= 1    Show getColor calls > $RGB value
+# DEBUG >= 2    Show test calls
+# DEBUG >= 3    Show all core functions calls
+# DEBUG >= 4    Show all functions calls
+# DEBUG >= 9    Show tap calls
+SHOW_DELTA=1
+# SHOW_DELTA = 0                    Show no debug
+# SHOW_DELTA = 1 && DEBUG >= 2      Show delta if color is not found
+DEFAULT_DELTA=2                                 # Default delta for colors
```

Comment:
I think I'll remove `SHOW_DELTA` because it's useless (I think we will always show the delta if in `DEBUG`)

The second one is used by default: `-d $DEFAULT_DELTA`, because it's easier for testing :)